imap_processing/ultra/l0/decom_ultra.py

```diff
@@ -77,7 +77,8 @@ def append_tof_params(
             data_dict[key].clear()
 
 
-def append_params(decom_data: dict, packet):
+def append_params(decom_data: dict, packet) -> None:  # type: ignore[no-untyped-def]
+    # Todo Update what packet type is.
```

Comment:
I think these `packet` parameter types are `space_packet_parser.parser.Packet`? (You may need to `import space_packet_parser` to get them working too)